code/modules/uplink/uplink_items/spy_unique.dm

```diff
@@ -11,6 +11,12 @@
 	// By default, these fall into easy-medium spy bounty loot pool
 	cost = SPY_LOWER_COST_THRESHOLD
 
+/datum/uplink_item/spy_unique/spy_knife
+	name = "Spy Knife"
+	desc = "Knife for real gentlemen."
+	item = /obj/item/knife/spy
+	cost = 1
```

Comment:
Keep it in the `SPY_LOWER_COST_THRESHOLD` range, so it can appear in easy or medium loot pools. 
```suggestion
```